docker/Dockerfile.helm-operator

```diff
@@ -1,8 +1,16 @@
-FROM alpine:3.9
+FROM debian:stable-slim
 
 WORKDIR /home/flux
 
-RUN apk add --no-cache openssh ca-certificates tini 'git>=2.12.0'
+RUN echo "deb http://deb.debian.org/debian stretch-backports main" | tee -a /etc/apt/sources.list.d/stretch-backports.list && \
+  echo "deb http://deb.debian.org/debian unstable main" | tee -a /etc/apt/sources.list.d/unstable.list && \
```

Comment:
Using unstable could lead to non working images, consider using testing instead. Also using `| tee` here doesn't end in any gain. a simple `>` will suffice